docs/monitoring/faq/webhooks.mdx

```diff
@@ -77,3 +77,116 @@ When delivering events to your webhook endpoint we follow these guidelines
 - If your endpoint returns a 5xx status code in less than 5 seconds we retry up to 2 times with exponential backoff.
 - If your endpoint returns a 4xx status code, we declare the delivery failed and do not retry.
 - If your endpoint returns any body do not read it
+
+## Example with Modal
+
+### Setup
+
+For an example of how to set this up, we will use [Modal](https://modal.com/).
+
+First, create a Modal account. Then, locally install the Modal SDK:
+
+```shell
+pip install modal
+```
+
```

Comment:
Here, I'd add:

To finish setting up your account, run the command
```bash
modal setup
```
and follow the instructions.